manning/manning-proposal-2020-07.md

```diff
@@ -43,7 +41,7 @@ We assume readers have some basic programming knowledge, and build on that to co
 All of this material has been used and refined in workshops,
```

Comment:
In the list of what we cover immediately above this line, "publishing in a reproducible way" might need to be removed in light of #415. Instead, we could say "documenting, packaging and releasing software" and that would be the second last bullet point (above "creating productive, inclusive teams" which comes after packaging in the #415 proposal).

Reply:
Done.